pulp_2to3_migration/app/plugin/rpm/repository.py

```diff
@@ -73,7 +73,8 @@ def migrate_to_pulp3(cls, pulp2distributor, repo_version):
             if pulp2_checksum_type:
                 checksum_types = {'metadata': pulp2_checksum_type,
                                   'package': pulp2_checksum_type}
-            publish(repo_version.pk, checksum_types=checksum_types)
+            sqlite = pulp2_config.get('generate_sqlite', False)
```

Comment:
This should work right?  It's the same distributor config as https://github.com/pulp/pulp_rpm/blob/2-master/plugins/pulp_rpm/plugins/distributors/yum/publish.py#L1181 ?